eZ/Publish/Core/Persistence/Cache/URLHandler.php

```diff
@@ -19,7 +20,7 @@
 class URLHandler implements URLHandlerInterface
```

Comment:
Please extend AbstractHandler and remove the duplicated logic

Reply:
Done 👍